packages/http/src/lib.ts

```diff
@@ -88,11 +88,10 @@ export const $lib = createTypeSpecLibrary({
         default: "`Content-Type` header ignored because there is no body.",
       },
     },
-    "no-routes": {
+    "no-service-found": {
```

Comment:
I guess the problem here is that if someone was previously suppressing `no-routes`, they'll now get a new diagnostic which might break their build.  Not really sure what to do about it.  @johanste?

Reply:
Ahhh, well that simplifies things :)